package-list.json

```diff
@@ -25,7 +24,7 @@
       "version": "0.1.0",
       "date": "2020-08-11",
       "desc": "STU 1 Ballot",
-      "path": "https://hl7.org/fhir/uv/smart-web-messaging/2020Sep",
+      "path": "http://hl7.org/fhir/uv/smart-web-messaging/2020Sep",
```

Comment:
Sad this can't be https -- but I am assuming you changed this back because it breaks something